src/main/scala/outwatch/dom/VDomModifier.scala

```diff
@@ -24,6 +42,17 @@ object Attribute {
   def apply(title: String, value: Attr.Value) = Attr(title, value)
 }
 
+
+sealed trait Hook[T] extends Property {
+  def observer: Observer[T]
+}
+
+// Attributes
+
+case object EmptyAttribute extends Attribute {
```

Comment:
can somehow not have an empty string as a title here?

Reply:
No, `EmptyAttribute`s are skipped and not passed to Snabbdom.